### PR TITLE
Add Proxy Page for Block Producers

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Adds a page on creating, removing, and executing transactions with proxies.

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [x] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

**Review by eshaben — COMMENTED** [review]
Nice job! I think we need to have a bit of a different mindset about these proxy docs versus Moonbeam's. For these docs, I think we need to cater them specifically to block producers. So instead of showing them how to set up a balances proxy, we should really lay out what exactly should be done to set up a proxy for their block producer account. I think block producers will need to use `Any` type, which is not necessarily ideal and kind of defeats the purpose haha... so I am double-checking on this

**Comment by eshaben** [comment]
Haha, I was looking at the right code, but totally overlooked the ability to call the session pallet with the staking proxy type. But that's great news because now we can use the staking proxy in this guide and show users how to use a proxy when setting up their block producer

File: node-operators/block-producers/operational-tasks/proxy-accounts.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+11. Click **Submit Transaction**
+
+![Execute a proxy transaction from the Extrinsics page of Polkadot.js Apps.](/images/node-operators/block-producers/operational-tasks/proxy/proxy-block-8.webp)
+
+A pop-up will appear for you to authorize and sign the transaction. Enter your password for the proxy account and click **Sign and Submit**. To confirm the delegation request, you'll need to execute the pending request after two sessions have passed. Please refer to the instructions for [executing pending requests](/node-operators/block-producers/onboarding/account-setup/#execute-pending-request){target=\_blank} for a step-by-step guide. After executing the pending request, you can [verify that your block producer is in the list of eligible candidates](/node-operators/block-producers/onboarding/account-setup/#verify){target=\_blank}.
+
+That's it! You've successfully executed a transaction using a proxy account on behalf of your primary account.
+
+## Removing a Proxy Account {: #removing-a-proxy-account }
```
Comment (eshaben) [inline node-operators/block-producers/operational-tasks/proxy-accounts.md:111]: ahh, ok. I'd say this section can just be in the generic Proxy Account page, and then in the Offboarding / Decommission page, we can add a note saying if you set up your block producer account with a proxy, check out the generic guide to learn how to remove it.

File: node-operators/block-producers/operational-tasks/proxy-accounts.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+## Introduction {: #introduction }
+
+Proxy accounts can be set up to perform a limited number of actions on behalf of primary accounts and are helpful for keeping the underlying accounts safe. As a block producer on Tanssi, it's a good idea to take advantage of proxy accounts to interact with the network in place of your block producer account. Your proxy account can act as a "hot wallet" to perform regular maintenance duties on behalf of your "cold wallet" block producer account. For added safety, you can regularly rotate the proxy account. 
+
+Proxy accounts can also help you implement the principle of least privilege for access control. For example, if you have multiple team members, you can give them the minimum level of access required to carry out their duties via a specific type of proxy account. For more background on Proxy accounts and how they work in Tanssi, be sure to check out the Guide to Proxy Accounts in the Learn Section.
+
+This tutorial will walk you through configuring a proxy account on the Tanssi Dancebox TestNet specifically for operations as a block producer. Then, it will demonstrate performing an example action using the newly created staking proxy.
```
[the remark this answers is not in the document]
Comment (eshaben) [reply node-operators/block-producers/operational-tasks/proxy-accounts.md:9]: after reading through this introduction, the only mention of the proxy type block producers need to use is this last sentence. i think we should explicitly state earlier on in the intro that the staking proxy is what they'll need, and from the staking proxy, they'll be able to submit/manage their self-delegation and map/remove their session keys

**Review by eshaben — COMMENTED** [review]
The example is great! Thank you!

However, I think it's important for the proxy account setup to be under the onboarding section, as the decision to setup a proxy would most likely happen at the time of onboarding. So just a little more shuffling around 🙂

File: node-operators/block-producers/operational-tasks/proxy-accounts.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
+# Setting up a Proxy Account
+
+## Introduction {: #introduction }
+
+Proxy accounts can be set up to perform a limited number of actions on behalf of primary accounts and are helpful for keeping the underlying accounts safe. As a block producer on Tanssi, it's a good idea to take advantage of proxy accounts to interact with the network in place of your block producer account. Your proxy account can act as a "hot wallet" to perform regular maintenance duties on behalf of your "cold wallet" block producer account. For added safety, you can regularly rotate the proxy account. 
+
+Proxy accounts can also help you implement the principle of least privilege for access control. For example, if you have multiple team members, you can give them the minimum level of access required to carry out their duties via a specific type of proxy account. For more background on Proxy accounts and how they work in Tanssi, be sure to check out the Guide to Proxy Accounts in the Learn Section.
```
[the remark this answers is not in the document]
Comment (themacexpert) [reply node-operators/block-producers/operational-tasks/proxy-accounts.md]: not yet but I'll add in the next pr

**Comment by eshaben** [comment]
The proxy accounts icon is a bit too thin:

Screenshot 2024-03-25 at 12 05 06 PM

**Comment by eshaben** [comment]
@themacexpert just merged the upgrade your node docs and now there are merge conflicts

File: node-operators/block-producers/.pages [hunk 4]
```diff
@@ -4,3 +4,4 @@ nav:
   - index.md
   - onboarding
   - offboarding
+  - operational-tasks
```
[the remark this answers is not in the document]
Comment (themacexpert) [reply node-operators/block-producers/.pages]: done!

**Comment by themacexpert** [comment]
> @themacexpert just merged the upgrade your node docs and now there are merge conflicts

ty, fixed!

File: node-operators/block-producers/.pages [hunk 5]
```diff
@@ -3,5 +3,6 @@ hide: false
 nav:
   - index.md
   - onboarding
+  - operational-tasks
   - offboarding
   - operational-tasks
```
Comment (eshaben) [inline node-operators/block-producers/.pages:8]:
```suggestion
```

File: node-operators/block-producers/operational-tasks/index.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Operational Tasks
-description: Learn how to manage your activities as a block producer in the Tanssi appchain protocol including managing your account, upgrading your node, and more.
+title: Block Producer Operational Tasks
+description: Learn how to manage your activities as a block producer in the Tanssi Appchain protocol, including managing your account, upgrading your node, and more.
```
Comment (eshaben) [inline node-operators/block-producers/operational-tasks/index.md:3]:
```suggestion
description: Learn how to manage your activities as a block producer in the Tanssi appchain protocol, including managing your account, upgrading your node, and more.
```

**Review by eshaben — COMMENTED** [review]
a couple of leftover things from the merge i think

**Review by eshaben — APPROVED** [review]
LGTM!! THANK YOU!! ✨ 🚀

**Pull request merged**